specs-go/config.go

```diff
@@ -22,11 +22,30 @@ type Spec struct {
 	Hooks Hooks `json:"hooks"`
 	// Annotations is an unstructured key value map that may be set by external tools to store and retrieve arbitrary metadata.
 	Annotations map[string]string `json:"annotations,omitempty"`
-
+	// KernelConfig specifies kernel-related configuration for
+	// virtual-machine-based runtimes.
+	KernelConfig KernelConfig `json:"kernelConfig"`
+	// ImagePath is the absolute path to the root filesystem image
+	// used by a virtual-machine-based runtime.
+	ImagePath string `json:"imagePath"`
 	// Linux is platform specific configuration for Linux based containers.
 	Linux Linux `json:"linux" platform:"linux"`
 }
 
+// Kernel contains information about the kernel to use for a virtual
+// machine.
```

Comment:
s/Kernel contains/KernelConfig contains/

and comments need to be single line.